api/chaos/experimentation/v1/experimentation.proto

```diff
@@ -68,7 +68,7 @@ message StopExperimentsResponse {
 service ExperimentsAPI {
   rpc CreateExperiment(CreateExperimentRequest) returns (CreateExperimentResponse) {
     option (google.api.http) = {
-      post : "/v1/experiments/create"
+      post : "/v1/experiment/create"
```

Comment:
still doesn't quite match, probably the guidelines need to be improved a bit more so they are clearer.

this would ideally be `/v1/chaos/experimentation/createExperiment` to match the package `clutch.chaos.experimentation.v1` and method name `CreateExperiment`

Reply:
I see, thanks for the explanation. Is `createExperiment` really preferred over `create-experiment`?

Reply:
yep, it's arbitrary i guess but we chose to go with camelcase throughout since it matches javascript naming conventions where these names are mostly used. google also uses camelcase throughout their APIs, and while we differ from their mappings in some ways (e.g. only using POST), that's one area we didn't feel was beneficial to diverge.

Reply:
Updated per your comments. Thanks for the detailed explanation.